src/applications/vaos/containers/AppointmentListsPage.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+
+import get from 'platform/utilities/data/get';
+import {
+  fetchConfirmedAppointments,
+  fetchPendingAppointments,
+} from '../actions/appointments';
+
+export class AppointmentListsPage extends Component {
+  componentDidMount() {
+    this.props.fetchConfirmedAppointments();
+    this.props.fetchPendingAppointments();
```

Comment:
I think we also need to focus on the `h1` element on this page after the page mounts (there's a `focusElement` helper in the UI utilities).